portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -65,6 +95,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(taskEntity);
 
+    response.sendRedirect("/index.html");
```

Comment:
per discussion in the meeting, we can probably skip the redirect here to allow the JS to add the sentence to the DOM

Reply:
This comment still applies. We can have the JS reload just this section